test/mocha.opts

```diff
@@ -2,6 +2,7 @@ test/helpers/*.js
 test
 --compilers js:babel-core/register
 --require ignore-styles
+--require babel-polyfill
```

Comment:
See https://babeljs.io/docs/usage/babel-register/ describing how the register compiler works if you're interested in a read!